runelite-client/src/main/java/net/runelite/client/plugins/zoom/ZoomPlugin.java

```diff
@@ -81,6 +83,7 @@ public void onScriptCallbackEvent(ScriptCallbackEvent event)
 			return;
 		}
 
+		int outerZoomLimit = Math.min(zoomConfig.outerLimit(), OUTER_ZOOM_LIMIT);
```

Comment:
This would probably break horribly if too negative.

Reply:
The zoom resets at around -2500, so it never becomes a problem for me at least. Even then it's limited at -1k